src/sfr.cpp

```diff
@@ -134,6 +140,9 @@ namespace sfr {
         float acc_x_average = 0.0;
         float acc_y_average = 0.0;
         float acc_z_average = 0.0;
+
+        imu_downlink_type imu_dlink_magid_type = imu_downlink_type::GAUSS_8;
+        float report[constants::sensor::collect * 5];
```

Comment:
Create value in the sfr for the size of the report. We might realize it needs to change between delivery/launch so it will be nice to have the ability to change that value from the ground.